src/interactive/repl.ts

```diff
@@ -381,13 +398,19 @@ export function activate(context: vscode.ExtensionContext, settings: settings.IS
 
     context.subscriptions.push(vscode.commands.registerCommand('language-julia.startREPL', startREPLCommand));
 
-    context.subscriptions.push(vscode.commands.registerCommand('language-julia.executeJuliaCodeInREPL', executeSelection));
+    context.subscriptions.push(vscode.commands.registerCommand('language-julia.executeJuliaCodeInREPL', evaluateBlockOrSelection));
+    context.subscriptions.push(vscode.commands.registerCommand('language-julia.executeJuliaCodeInREPLAndMove', () => evaluateBlockOrSelection(true)));
+
+    context.subscriptions.push(vscode.commands.registerCommand('language-julia.executeJuliaCellInREPL', executeJuliaCellInRepl));
+    context.subscriptions.push(vscode.commands.registerCommand('language-julia.executeJuliaCellInREPLAndMove', () => executeJuliaCellInRepl(true)));
+
+    // context.subscriptions.push(vscode.commands.registerCommand('language-julia.executeJuliaCodeInREPL', executeSelection));
```

Comment:
That's because [`evaluateBlockOrSelection`](
https://github.com/julia-vscode/julia-vscode/blob/3ae906bffe633e39daf2b65e1081836196578cc0/src/interactive/repl.ts#L297-L329) will automatically handles both cases with or without selections.
It's better to not require them to use two different commands (and so probably different keybindings), no ?